packages/client/src/object/aggregate.test.ts

```diff
@@ -133,53 +144,58 @@ const mockOntology = {
 type mockOntology = typeof mockOntology;
 interface MockOntology extends mockOntology {}
 
-describe("aggregate", () => {
-  it("works", async () => {
-    const mockFetch: Mock = vi.fn();
+let mockFetch: Mock;
+let clientCtx: MinimalClient;
+
+beforeEach(() => {
```

Comment:
If you don't modify anything in here (fetch function, client etc.), you can just use `beforeAll` which is only called once before everything